src/shared/types/ProductTypes.ts

```diff
@@ -1,12 +1,17 @@
+export type ProductSectionUnion = "new" | "popular";
```

Comment:
Are u going to use this type somewhere else ProductSection component? If not put it localy inside ProductSection.